back/strapi/scripts/import.js

```diff
@@ -0,0 +1,37 @@
+const fs = require("fs");
+const path = require("path");
+const fetch = require("node-fetch");
+
+const SEED_DIR = "seeds";
+
+const API_URL =
+  "http://backoffice-preprod-les1000jours.dev2.fabrique.social.gouv.fr";
+
+const plural = (str) => `${str}s`.replace(/ss$/, "s");
+
+const models = ["parcours", "thematique", "etape", "evenement", "article"];
+
+const main = async () => {
+  for (const model of models) {
```

Comment:
`models.forEach` ? Je préfère vraiment le fonctionnel ^^

Reply:
L'avantage du `for ... of` c'est qu'il gère l'async, et ici je dois faire les choses séquentiellement (ex : les étapes doivent être ajoutées avant les articles, etc.).

On pourrait le faire comme ceci :
```js
models.reduce(async (promise, model) => {
  await promise

  // ...
}, Promise.resolve())
```
mais c'est beaucoup plus obscure.

Reply:
Gardons for of alors ;)